ci/images/ci-runner/hack/bin/run-plnsvc-test.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+echo "Run pipeline-service tests..."
+if kubectl -n default exec pod/ci-runner -- \
+    sh -c "/workspace/sidecar/bin/plnsvc_test.sh"; then
```

Comment:
can we move the content of `/workspace/sidecar/bin/plnsvc_test.sh` to here? then we could delete ``/workspace/sidecar/bin/plnsvc_test.sh``

Reply:
The sidecar is the place where the version of the code that is tested is located.
If we run the tests from the TaskRun pod, then the pod would need to start switching between branches, which causes issues when the CI scripts are modified. E.g. if we change the `pipeline-service-setup` Task to a new step that references a new script, then the task would fail after we've checked out the target version that does not have the script.